src/storage-client/src/controller/statistics.rs

```diff
@@ -75,11 +75,7 @@ pub(super) fn spawn_statistics_scraper(
 
                                 packer.push(Datum::from(src_id.to_string().as_str()));
                                 packer.push(Datum::from(u64::cast_from(*worker_id)));
-                                packer.push(Datum::from(stats.snapshot_committed));
-                                packer.push(Datum::from(stats.messages_received));
-                                packer.push(Datum::from(stats.updates_staged));
-                                packer.push(Datum::from(stats.updates_committed));
-                                packer.push(Datum::from(stats.bytes_received));
+                                stats.pack(packer);
```

Comment:
I can't comment up above, but the loop variables should probably not be `src_id`/`sources` anymore?